recipe/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "azureml_mlflow" %}
```

Comment:
@boldorider4 The rule is to try to stick with the original name on PyPI. For example, see https://pypi.org/project/azureml-mlflow/. With you you can install both `azureml-mlflow` and `azureml_mlflow` because the ecosystem normalizes names (see https://packaging.python.org/en/latest/specifications/name-normalization/). But we don't need to user underscores in conda.

Reply:
I think I can pull the old trick with the multi-output package so both names are resolved.

Reply:
We don't need to support both. Keep the original name(`azureml-mlflow`). Adding both just adds unnecessary maintenance.